mob_test.go

```diff
@@ -822,7 +826,12 @@ func TestStartNextStay_DoNotWriteLastModifiedFileInCommit_WhenFileIsMoved(t *tes
 
 	start(configuration)
 	createDirectory(t, "dir")
-	run(t, "mv", workingDir+"/"+"file1.txt", workingDir+"/dir/"+"file1.txt")
+	oldPath := filepath.Join(workingDir, "file1.txt")
+	newPath := filepath.Join(workingDir, "dir", "file1.txt")
+	err := os.Rename(oldPath, newPath)
+	if err != nil {
+		failWithFailure(t, "no error", fmt.Sprintf("error %v occured moving %s to %s", err, oldPath, newPath))
+	}
```

Comment:
We should extract the file handling into a function as this is not essential for the test.